tests/test_executor.py

```diff
@@ -91,7 +94,37 @@ def to_csv(expression):
                     ),
                 )
             ):
-                with self.subTest(f"tpch-h {i + 1}"):
+                with self.subTest(f"tpc-h {i + 1}"):
+                    sql, _ = self.sqls[i]
```

Comment:
this can probably be a helper method so that we can reuse both right? perhaps the whole test could be a helper method

Reply:
Done